server/events.lua

```diff
@@ -168,18 +168,16 @@ RegisterNetEvent('QBCore:Server:UseItem', function(item)
     QBCore.Functions.UseItem(src, item)
 end)
 
-RegisterNetEvent('QBCore:Server:RemoveItem', function(itemName, amount, slot)
+-- This event is exploitable and should not be used. Will be deprecated soon. function(itemName, amount, slot)
```

Comment:
```suggestion
-- This event is exploitable and should not be used. It has been deprecated. function(itemName, amount, slot)
```
It _is_ deprecated

Reply:
Should add `Will be removed soon` instead

Reply:
> Should add `Will be removed soon` instead

Updated it, should be better like this